purchase_work_acceptance/models/work_acceptance.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class WorkAcceptance(models.Model):
+    _name = 'work.acceptance'
+    _inherit = ['mail.thread', 'mail.activity.mixin', 'portal.mixin']
+    _description = 'Work Acceptance'
+    _order = 'id desc'
+
+    name = fields.Char(
+        required=True,
+        index=True,
+        copy=False,
+        default='New'
+    )
+    date_due = fields.Datetime(
+        string='Due Date',
+        required=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]},
+    )
+    date_receive = fields.Datetime(
+        string='Received Date',
+        default=fields.Datetime.now,
+        required=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]},
+    )
+    date_accept = fields.Datetime(
+        string='Accepted Date',
+        readonly=True,
+    )
+    invoice_ref = fields.Char(
+        string='Invoice Reference',
+        copy=False,
+    )
+    partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Vendor',
+        required=True,
+        change_default=True,
+        track_visibility='always',
+        readonly=True,
+        states={'draft': [('readonly', False)]},
+    )
+    responsible_id = fields.Many2one(
+        comodel_name='hr.employee',
```

Comment:
Why employee? You should use res.users or res.partner in order to not make necessary hr

Reply:
Edited: https://github.com/OCA/purchase-workflow/pull/774/commits/aa231bbdd8d25ed7e48a94eeacb137b06599d07d